lib/OpenQA/Task/Job/FinalizeResults.pm

```diff
@@ -53,7 +53,8 @@ sub _finalize_results {
         my $timeout      = $ENV{OPENQA_JOB_DONE_HOOK_TIMEOUT}      // '5m';
         my $kill_timeout = $ENV{OPENQA_JOB_DONE_HOOK_KILL_TIMEOUT} // '30s';
         my $ret          = _done_hook_new_issue($openqa_job, $hook, $timeout, $kill_timeout);
-        $minion_job->note(hook_cmd => $hook, hook_result => $ret);
+        my $rc           = $?;
```

Comment:
Does the `$?` actually represent the exit code of the hook command? How did you test? Please describe in the commit

Reply:
I added a test for hook_rc in 10-jobs.t now.
`$?` contains the exit code of the last command, e.g. `qx{}` or `system()`.
I rearranged it a bit now, to make sure `$?` is used directly after the call.

Reply:
I also added an error case to check if we get the correct exit code

Reply:
very nice. The test explains it well enough